enterprise/api/renderers.py

```diff
@@ -8,13 +8,18 @@
 from rest_framework.settings import api_settings
 
 
-class IgnoreClientContentNegotiation(DefaultContentNegotiation):
+class CustomContentNegotiation(DefaultContentNegotiation):
     """
-    Class for client content negotiation.
+    DRF was ignoring the order in render_classes, may be due to djangorestframework-xml library.
+    By Default, It was giving precedence to xml by default so we need to override the select_renderer method.1
     """
 
     def select_renderer(self, request, renderers, format_suffix):  # pylint: disable=signature-differs
-        # Allow URL style format override.  eg. "?format=json or http://example.com/v1/organizations.xml/
+        """
+        Allow URL style format override. Either you can specify the querystring parameter (/?format=json
+        or /?format=xml) or XML/JSON suffix at end of URL e.g. shttp://example.com/v1/organizations.xml/
```

Comment:
nit: `or XML/JSON suffix at end of URL e.g. shttp://example.com/v1/organizations.xml/` -> 
`or add XML/JSON suffix at end of URL, e.g. http://example.com/v1/organizations.xml`